R/transformToBista.R

```diff
@@ -1,4 +1,16 @@
-transformToBista <- function ( equatingList, refPop, cuts, weights = NULL, defaultM = 500, defaultSD = 100, q3bound = .20, roman = FALSE, vera = TRUE, idVarName = NULL, years = NULL ) {
+transformToBista <- function(equatingList, refPop, cuts, weights = NULL,
+                             defaultM = 500, defaultSD = 100, q3bound = .20,
+                             roman = FALSE, vera = TRUE, idVarName = NULL,
+                             years = NULL){
+### checks ---------------------------------------------------------------------
+
+  checkmate::assert_data_frame(weights, ncols = 2, null.ok = TRUE)
+  lapply(c(defaultM, defaultSD, q3bound), checkmate::assert_numeric, len = 1, lower = 0)
+  checkmate::assert_numeric(years, len = 2, null.ok = TRUE)
```

Comment:
To be totally precise, you could add `unique=TRUE` here, i.e.  `checkmate::assert_numeric(years, len = 2, null.ok = TRUE, unique=TRUE)`